web/styles/settings.css

```diff
@@ -1447,14 +1452,13 @@ label.preferences-radio-choice-label {
             text-align: center;
             font-size: 1.1em;
             line-height: 1;
-            margin: 15px;
```

Comment:
The margin isn't necessary anymore because the contents are centered with flex. Previously they were un-centeredly placed with margin.